ui-components/dialog.md

```diff
@@ -144,7 +144,7 @@ A **button** is a clickable component that can contain text or an icon. There ar
 
 | Name | Type | Requirement | Description |
 | ---- | ---- | ----------- | ----------- |
-| type | `'button'` | required | The component type. Must be `'button'`. |
+| type | `'submit'` or `'cancel'` or `'custom'` | required | Must be `'submit'` or `'cancel'` or `'custom'` based on the type of callback function that should be invoked when the button is clicked. |
```

Comment:
`Must be 'submit', 'cancel' or 'custom'`? The double "or" doesn't seem right to me.

Reply:
Fixed